ts/WoltLabSuite/Core/Ui/Comment/Add.ts

```diff
@@ -10,6 +10,7 @@
  * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
  * @module  WoltLabSuite/Core/Ui/Comment/Add
  * @woltlabExcludeBundle tiny
+ * @deprecated 6.0 use `WoltLabSuite/Core/Component/Comment/Add` instead
```

Comment:
Doesn’t it make more sense to remove these entirely? They serve no purpose anymore and I consider those to be part of the internal API.